winit/src/application.rs

```diff
@@ -179,7 +178,7 @@ where
 
     let mut context = task::Context::from_waker(task::noop_waker_ref());
 
-    event_loop.run_return(move |event, _, control_flow| {
+    event_loop.run(move |event, _, control_flow| {
```

Comment:
@hecrj do you think this would break anything? It simplifies the code a lot and the `winit` docs says about `run_return` "You are strongly encouraged to use run, unless the use of this is absolutely necessary."